website/README.md

```diff
@@ -5,21 +5,26 @@ This website is built using [Docusaurus 2](https://docusaurus.io/), a modern sta
 ## Installation
 
 ```console
-yarn install
+npm install
```

Comment:
yarn was never used by devs judgeing by the `package-lock.json` (instead of `yarn.lock`)
npm is also what's used in our workflows.